lib/rucio/client/baseclient.py

```diff
@@ -804,7 +826,7 @@ def __get_token_saml(self):
         self.auth_token = result.headers['X-Rucio-Auth-Token']
         return True
 
-    def __get_token(self):
+    def __get_token(self) -> Optional[bool]:
```

Comment:
```suggestion
    def __get_token(self) -> None:
```

This function doesn't seem to return anything